Nef_S2/include/CGAL/Nef_S2/Sphere_circle.h

```diff
@@ -78,15 +80,20 @@ create any great circle that contains $p$ and $q$.}*/
   }
 }
 
- Sphere_circle(const Plane_3& h) : Base(h)
+Sphere_circle(const Plane_3& h) : Base(h)
 /*{\Mcreate creates the circle of $S_2$ corresponding to the plane
 |h|. If |h| does not contain the origin, then |\Mvar| becomes the
 circle parallel to |h| containing the origin.}*/
 {
-  if(h.d() != 0) *this = Plane_3(h.a(),h.b(),h.c(),RT(0));
+  if(h.d() != 0)
```

Comment:
While we are at it, maybe use `is_zero()`here.

Reply:
I initially investigated `!is_zero()` I think they ultimately use the same filtering. But I will add to better show intention.